docker/dev/php/Dockerfile

```
@@ -1,4 +1,4 @@
-FROM php:7.4-cli-alpine3.15
+FROM php:8.2-cli-alpine3.16
```

```suggestion
FROM php:7.4-cli-alpine3.16
```
i kept 7.4 on purpose to see if backward compatibility was broken somewhere :smile:

---

returned it back